src/main/java/org/jitsi/utils/queue/PacketQueue.java

```diff
@@ -129,8 +131,8 @@ public PacketQueue(
         {
             enableStatistics = enableStatisticsDefault;
         }
-        queueStatistics
-            = enableStatistics ? new QueueStatistics() : null;
+        observer
```

Comment:
Maybe take in an optional `Observer` instead of `enableStatistics`?  It seems like this bit should be external, but maybe I'm not fully following.

Reply:
This also feels a bit weird given there's a `setObserver` call as well, seems like it should just be an external class creating the observer and adding/setting it?

Reply:
That's what the old (last year's) PR did - but it resulted in a bunch of duplicated code for all the observers, which seemed unnecessary, so I thought adding the shortcut for this normal case was useful.

Reply:
So a caller can either set its own observer which it would access via the observer directly, or tell `PacketQueue` to enable statistics, where it creates its own observer and exposes stats for?  Seems like we'd only need one of those use cases.

Reply:
Yeah, I'll remove the `setObserver` case.